README.md

```diff
@@ -13,10 +13,7 @@
 
 
 > ## Deprecation Warning:
-> **All TBomb versions below v2.0 will no longer work after 14-11-2020.**  
-**All TBomb users need to update to v2.0 ASAP**
-
-**Due to overuse of script, a bunch APIs have been taken offline. It is okay if you do not receive all the messages.**
+> **Due to overuse of script, a bunch APIs have been taken offline. It is okay if you do not receive all the messages.**
```

Comment:
Please correct this to `Due to the overuse of script, a bunch of APIs have been taken offline. It is okay if you do not receive all the messages.`

Reply:
Welp, how do i am do that? In my Branch?